client/my-sites/customer-home/locations/secondary/index.jsx

```diff
@@ -23,16 +26,39 @@ const cardComponents = {
 };
 
 const Secondary = ( { cards } ) => {
+	if ( ! cards ) {
+		return null;
+	}
+	const allPossibleEducationalCards = [
+		'home-education-gutenberg',
+		'home-education-free-photo-library',
+	];
+	const educationalCards = cards.filter( card => allPossibleEducationalCards.includes( card ) );
+	let skipEducationalCard = false;
 	return (
 		<>
-			{ cards &&
-				cards.map(
-					( card, index ) =>
-						cardComponents[ card ] &&
-						React.createElement( cardComponents[ card ], {
-							key: index,
-						} )
-				) }
+			{ cards.map( ( card, index ) => {
+				if ( ! cardComponents[ card ] ) {
+					return null;
+				}
+				if ( educationalCards.includes( card ) ) {
+					if ( skipEducationalCard ) {
+						return null;
+					}
+
+					skipEducationalCard = true;
+					return (
+						<DotPager>
+							{ educationalCards.map( educationalCard =>
+								React.createElement( cardComponents[ educationalCard ] )
```

Comment:
I think needs a key:

<img width="1069" alt="Screen Shot 2020-04-16 at 10 17 08 AM" src="https://user-images.githubusercontent.com/349751/79486428-897d4b80-7fcb-11ea-8bdf-88a048985d36.png">

Reply:
💯

Reply:
@kwight feel free to add on a change here, or let me know if you don't have time. It would be lovely to get this out to folks.